alf/environments/suite_carla.py

```diff
@@ -777,15 +806,16 @@ def action_spec(self):
         """Get the action spec.
 
         The action is a 4-D vector of [throttle, steer, brake, reverse], where
-        throttle is in [0.0, 1.0], steer is in [-1.0, 1.0], brake is in
-        [0.0, 1.0], and reverse is interpreted as a boolean value
-        with values greater than 0.5 corrsponding to True.
+        throttle is in [-1.0, 1.0] (negative value is same as zero), steer is in
+        [-1.0, 1.0], brake is in [-1.0, 1.0] (negative value is same as zero),
```

Comment:
You are right, merging throttle and brake can make it easier to learn. But I think in some rare situations both needs to be applied. For example, starting the car on a steep slope, you may need to apply throttle and brake at the same time so that the car won't go backward. action repeat can make it easier to learn too, but may not be optimal in the end.

Reply:
> Remove this comment, but instead, point to action_desc()?

@hnyu suggested to add the comment in the previous review.

Reply:
> Remove this comment, but instead, point to action_desc()?

Yeah we have to consider that docstring is to be displayed on the webpage. Not everyone wants to look at the internal code.

Reply:
> > Remove this comment, but instead, point to action_desc()?
> 
> @hnyu suggested to add the comment in the previous review.

😅 maybe remove the details but just point out that it's keyboard controlled?  Every time it is run, action_desc will be displayed in the terminal, so it's fine not to show the details in the doc?

Reply:
It's convenient to just read the doc. It takes quite a bit of effort to run this.